tests/kernel/common/src/irq_offload.c

```diff
@@ -121,11 +121,13 @@ __no_optimization void test_nop(void)
 	/* do 2 nop instructions to cost cycles */
 	arch_nop();
 	arch_nop();
-#elif defined(CONFIG_ARMV6_M_ARMV8_M_BASELINE)
+#elif defined(CONFIG_ARMV6_M_ARMV8_M_BASELINE) || \
+	defined(CONFIG_ARMV7_M_ARMV8_M_MAINLINE)
 	/* do 4 nop instructions to cost cycles */
```

Comment:
Update the comment to say 5.  There's one arch_nop() before the ifdef.

Reply:
well.. depends on how you count :-).
Updated comments here and there to make things clear.